<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mianliao_chuku_edit</class>
 <widget class="QDialog" name="mianliao_chuku_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>531</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="Label_15">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>入库单号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LineEdit_ruku_id_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="Label_16">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>日期</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="Label_19">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>米数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="LineEdit_mishu_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="Label_20">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>公斤</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LineEdit_gongjin_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="Label_21">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>卷数</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="LineEdit_juanshu_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="Label_17">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>订单号</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="LineEdit_dingdanhao_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="Label_18">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>领取人</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="LineEdit_lingquren_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="Label_22">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>目标去向</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="LineEdit_targetto_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="Label_23">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>备注</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="LineEdit_comment_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="palette">
       <palette>
        <active/>
        <inactive/>
        <disabled/>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>确认修改出库</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="dateEdit_date_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>mianliao_chuku_edit</receiver>
   <slot>MianLiaoChukuSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>MianLiaoChukuSlot()</slot>
 </slots>
</ui>
